includes/polling/os/hiveos-wireless.inc.php

```diff
@@ -23,19 +23,10 @@
  * @author     https://github.com/theherodied/
  */
 
-if (preg_match('/^(.+?),/', $device['sysDescr'], $hardware)) {
-    $hardware = $store[1];
-}
-$hardware = trim(snmp_get($device, '.1.3.6.1.4.1.4413.1.1.1.1.1.3.0', '-Ovq'), '"');
-$version = trim(snmp_get($device, '.1.3.6.1.4.1.4413.1.1.1.1.1.13.0', '-Ovq'), '"');
-$serial = trim(snmp_get($device, '.1.3.6.1.4.1.4413.1.1.1.1.1.4.0', '-Ovq'), '"');
-
 $apmodel = snmp_get($device, 'ahDeviceMode.0', '-Ovq', 'AH-SYSTEM-MIB');
-if ($apmodel == 'AP130' || $apmodel == 'AP250' || $apmodel == 'AP550') {
     $data = snmp_get_multi_oid($device, 'ahSystemSerial.0 ahDeviceMode.0 ahFirmwareVersion.0', '-OQUs', 'AH-SYSTEM-MIB');
```

Comment:
Can you remove all of these spaces now so it's formatted correctly.

Reply:
Like that?

Reply:
@laf I removed the spaces but codeclimate doesn't like it.